pages/character/ranking.go

```diff
@@ -21,7 +20,7 @@ func Ranking(ctx aero.Context) error {
 	response := struct {
 		Rank       int     `json:"rank"`
 		Percentile float64 `json:"percentile"`
-	}{}
+	}
```

Comment:
The bracket is required to initialize an instance of the previously declared struct.